### ISSUE TITLE
Instant LvL 70

### ISSUE THREAD

**Issue by ndjohnson1977** [dalaranwow/dalaran-wow#4335]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)

I attempted to lvl a character to 70.  That's why I came here as I didn't want to grind out yet another character and I am not happy with the current wow.  I'd love to get back into wotlk.  When I selected the option to level a character to 70 my character did not show up as an option.

I am able to see my character in other areas such as the faction change or appearance change.

**Expected behaviour**: Tell us what should happen instead

I believe the character should populate or be available in a drop down selection.

**Steps to reproduce the problem**:

1. 
2. 
3. 

**Include proofs for this behaviour**

![image](https://cloud.githubusercontent.com/assets/25380279/22360757/7f6f1bfc-e418-11e6-938e-25e33f9d29ae.png)

![image](https://cloud.githubusercontent.com/assets/25380279/22360759/8c7d9bde-e418-11e6-824e-24bb1974bf7b.png)

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like http://wotlk.openwow.com or http://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by Edzero** [comment]
The issuetracker is for ingame/serverside issues only. 

For assistance with such issues please use the forum, email or write an in-game ticket.

If your character is on Kirin Tor that may be the reason (I am not a GM though so don't quote me on that).
